ansible_base/lib/dynamic_config/dynamic_settings.py

```diff
@@ -26,6 +26,12 @@
     REST_FRAMEWORK = {}
 
 
+try:
+    ANSIBLE_BASE_SERVICE_PREFIX
+except NameError:
+    ANSIBLE_BASE_SERVICE_PREFIX = 'local'
```

Comment:
Would it be an issue if there were "multiple" locals? i.e. should this be a UUID or some other unique string?

Reply:
Anything using resource registry or centralized RBAC should set this. This will prefix the type like "local.inventory", and I hate the idea of a UUID there. My hot take here is that "local" works for something that is absolutely not going to be doing any clustering with another system via DAB stuff.